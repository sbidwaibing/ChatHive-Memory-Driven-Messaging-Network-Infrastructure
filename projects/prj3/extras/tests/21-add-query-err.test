+ @john room-21 #sqlite #db
sqlite is cool
.
? room-21
.
? room-21 #x
.
? xx #db
.
