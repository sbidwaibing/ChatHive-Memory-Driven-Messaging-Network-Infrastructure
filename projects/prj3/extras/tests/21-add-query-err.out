ok
ok
YYYY-MM-DDThh:mm:ss.ttt
@john room-21 #db #sqlite
sqlite is cool
err BAD_TOPIC
err BAD_ROOM
